.access.yml

```diff
@@ -5,36 +5,39 @@ policies:
     roles:
       - admin
       - maintain
+      - write
     pull_request: false
     users_from_json: https://raw.githubusercontent.com/Quansight/open-gpu-server/main/access/conda-forge-users.json
   - id: conda-smithy-feedstock-policy
     repo: conda-smithy-feedstock
     roles:
       - admin
       - maintain
+      - write
     pull_request: false
     users_from_json: https://raw.githubusercontent.com/Quansight/open-gpu-server/main/access/conda-forge-users.json
   - id: conda-forge-ci-setup-feedstock-policy
     repo: conda-forge-ci-setup-feedstock
     roles:
       - admin
       - maintain
+      - write
     pull_request: false
     users_from_json: https://raw.githubusercontent.com/Quansight/open-gpu-server/main/access/conda-forge-users.json
   - id: libmagma-feedstock-policy
     repo: libmagma-feedstock
     roles:
       - admin
       - maintain
+      - write
     pull_request: true
     users_from_json: https://raw.githubusercontent.com/Quansight/open-gpu-server/main/access/conda-forge-users.json
   - id: pytorch-cpu-feedstock-policy
     repo: pytorch-cpu-feedstock
     roles:
       - admin
       - maintain
-      - admin
-      - maintain
```

Comment:
I'll fix this duplication issue soon.

Reply:
Thanks for noticing!

Reply:
Isuru created an issue while ago: https://github.com/AktechLabs/cirun-py/issues/3